metashare/api/models.py

```diff
@@ -129,6 +129,22 @@ def __str__(self):
         return self.url
 
 
+class ProjectSlug(AbstractSlug):
+    parent = models.ForeignKey(
+        "Project", on_delete=models.PROTECT, related_name="slugs"
+    )
+
+
+class Project(mixins.HashIdMixin, mixins.TimestampsMixin, SlugMixin, models.Model):
+    name = models.CharField(max_length=50, unique=True)
+    description = MarkdownField(blank=True, property_suffix="_markdown")
+    pr_url = models.URLField(unique=True)
```

Comment:
@wlonk Projects will always have a `branch_url` (not sure if we need to store that or not). I'm not clear on whether we're also creating a PR immediately when the Project is created (and then just marking it "ready for review" somehow on the "capture changes" action), or waiting until "capture changes" to create the PR at all.

Reply:
Wait until changes are captured, I think.

Should we just store a PR id and synthesize the rest of the URL? If we move a repository to a different github organization, I want to have to update the repository URL in exactly one place in MetaShare.